packages/components/src/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import { Meta } from '@storybook/react'
+import { DropdownMenu, IDropdownPosition } from './Dropdown'
+import React from 'react'
+import { PrimaryButton } from '../../src/buttons'
+
+const Template = (args: {
+  position: IDropdownPosition
+  offset_x: number
+  offset_y: number
+}) => (
+  <div
+    style={{
+      position: 'relative',
+      width: '100%',
+      height: '400px',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center'
+    }}
+  >
+    <DropdownMenu>
+      <DropdownMenu.Trigger>
+        <PrimaryButton>Click Me</PrimaryButton>
+      </DropdownMenu.Trigger>
+      <DropdownMenu.Content {...args}>
+        <div>
+          <DropdownMenu.Label>City</DropdownMenu.Label>
+          <DropdownMenu.Separator />
+          <DropdownMenu.Item
+            onClick={() => alert('Dhaka is the capital of Bangladesh')}
+          >
+            Dhaka
+          </DropdownMenu.Item>
+          <DropdownMenu.Item
+            onClick={() => alert('Helsinki is the capital of Finland')}
+          >
+            Helsinki
+          </DropdownMenu.Item>
+        </div>
+        <DropdownMenu.Separator weight={2} />
+        <div>
```

Comment:
I think it's to group the items but the `div` seems a bit random. Should we use something named for it or is this a API choice that is done in other libraries as well?

Reply:
We don't actually have any logic for let's say: `<ItemGroup>`. We can always add it later if it is required. I'll remove the `<div>`s as they are not doing anything.